### PR TITLE
Garrison SCOMline

### PR DESCRIPTION
**Pull request by Onutsio**
## About The Pull Request

This PR does the following:

Adds in a new SCOMstone and serfstone variant, the Crownstone and Houndstone respectively. 

Crownstones can swap between garrison line and general line by using them in hand, and can broadcast over either. Garrison line only broadcasts to houndstones, other crownstones, and SCOMs with their garrison line opened(more on that later)

Houndstones are serfstones that also hear the garrison line.

Garrison members(MAA, Sergeant, Wardens, RG, Squires) can now MMB SCOMs to open a garrison line. This makes the SCOM only broadcast to houndstones, crownstones, and other SCOMs with their garrison line opened. Make sure to close it back after you leave, or you'll leave a fun open garrison comms line to the public.

So, who gets crownstones, and who gets houndstones?

The Duke gets 2 crownstones which spawn in his room now, instead of the regular scomstones. One for him, one for the consort. The Officers(Sergeant and Guard Captain) get a crownstone. Marshall gets a crownstone as well

Wardens, MAA, RG, Squires, and the dungeoneer get houndstones, with which to receive orders and such by.

This PR also strips signal horns off of everyone. These were quite bad, I'll touch on that some more in the next section.

Additionally, adds a hear range variable to scomstones(if anyone wants to have the shorter hear range/louder one) in the future. And a message received sound variable. Just kinda helpful for people who may want to make different scomstones later on, and I used it myself so.

## Why It's Good For The Game

I'm a little unsure about this still. It might not be? My general idea is that Garrison members *tend* to be better behaved than the adventurer swarms and such when it comes to antagonist situations. Limiting their communications to eachother would let *them* handle problems and not key in the entire town to kill the bad guy like signal horns do.

Additionally, signal horns were always a 'I whip this out, blow it, prepare for 40 guys to come or to get lost because signal horns are hard to follow' button. Now, garrison members have more *coherent* information, but regular garrison members can't just call for help from anywhere and require a stationary scom. Antagonistic forces can KNOW that the OFFICER can call for reinforcements at any time. Kinda like a vidya game, right? Usual trope of the OFFICER being the guy who can call for help.

Also, these are 100% stealable/usable by antags/etc for information. Though, you'll have to pick it off a garrison member(unless they just leave it around for some reason) or rob them for it. A higher risk reward in tapping into garrison communications.

### CODE REVIEW

**Comment by kittysmooch** [comment]
good idea. my only thoughts are that there should probably be a way to craft these if they get lost, stolen, qdel'd by acid or whatever, or for the captain's levies, and that squires should probably get access to the scomline even if they dont get roundstart rings since a lot of lowpop rounds have basically no garrison except two squires and a random captain or whatever

**Comment by Onutsio** [comment]
> good idea. my only thoughts are that there should probably be a way to craft these if they get lost, stolen, qdel'd by acid or whatever, or for the captain's levies, and that squires should probably get access to the scomline even if they dont get roundstart rings since a lot of lowpop rounds have basically no garrison except two squires and a random captain or whatever

I think for now I want them to maintain being roundstart only. I *really* dont want a very rich keep to start handing these out to every merc/adventurer they recruit. If you lose 'em, you lose 'em. I will however give squires the serfstone variant and the ability to open scomlines on SCOMs.

**Comment by radarseas** [comment]
I think its best to make these obtainable by a steward import, so its in crown hands if to order it in. Solves most issues, _if someone steals the key to import off the nerve master_, they've **probably stolen the entire ecomony too** and probably will need it to survive the onslaught.

`I'd add a trait to handle who can't and can instead of checking guard ones and job, so people conscripted into the town watch, hired garrison mercs, adminbus and royals that lose their crownstones can still communicate to the garrison.`

**Comment by Onutsio** [comment]
I'm mostly just keeping these on being roundstart items for garrison as a reward for actually playing garrison. My fear with the nerve master isnt actually antagonists purchasing them, but rather rich keeps just doling them out like candy to random mercs and such to participate in the deathball with sharper communications.  Same thing with the concept of letting others use the garrison scomlines.

**Comment by Lyroy** [comment]
Dissuading the only source of reliable employment for mercenaries (the keep) from hiring them because they're going to be lacking a crucial piece of equipment is just another way to push mercenaries to be adventurer+ and go do dungeons.

These should be importable 100%.

**Comment by kittysmooch** [comment]
:pensive:

**Comment by Onutsio** [comment]
K my break is over + merge conflicts good 2 go
